mmv1/products/dlp/api.yaml

```diff
@@ -1328,4 +1330,267 @@ objects:
+                                      description: Seconds of minutes of the time. Must normally be from 0 to 59. An API may allow the value 60 if it allows leap-seconds.
+                                    - !ruby/object:Api::Type::Integer
+                                      name: nanos
+                                      description: Fractions of seconds in nanoseconds. Must be from 0 to 999,999,999.
+                                - !ruby/object:Api::Type::NestedObject
+                                  name: dateValue
+                                  description: Represents a whole or partial calendar date.
+                                  properties:
+                                    - !ruby/object:Api::Type::Integer
+                                      name: year
+                                      description: Year of the date. Must be from 1 to 9999, or 0 to specify a date without a year.
+                                    - !ruby/object:Api::Type::Integer
+                                      name: month
+                                      description: Month of a year. Must be from 1 to 12, or 0 to specify a year without a month and day.
+                                    - !ruby/object:Api::Type::Integer
+                                      name: day
+                                      description: Day of a month. Must be from 1 to 31 and valid for the year and month, or 0 to specify a year by itself or a year and month where the day isn't significant.
+                                - !ruby/object:Api::Type::Enum
+                                  name: dayOfWeekValue
+                                  description: Represents a day of the week.
+                                  values:
+                                    - :MONDAY
+                                    - :TUESDAY
+                                    - :WEDNESDAY
+                                    - :THURSDAY
+                                    - :FRIDAY
+                                    - :SATURDAY
+                                    - :SUNDAY
+                        - !ruby/object:Api::Type::NestedObject
+                          name: redactConfig
+                          description: |
+                            Redact a given value. For example, if used with an InfoTypeTransformation transforming PHONE_NUMBER, and input 'My phone number is 206-555-0123', the output would be 'My phone number is '.
+                          properties: [] # Meant to be an empty object with no properties - see here : https://cloud.google.com/dlp/docs/reference/rest/v2/projects.deidentifyTemplates#redactconfig
+                          # The fields below allow POST requests to include the "redactConfig" transformation in the payload
+                          # The trade-off is that POST requests specifying another transformation type will include will include "redactConfig" : null in the payload
```

Comment:
could you add a link to the bug here?

Reply:
Done in [ecf83d6](https://github.com/GoogleCloudPlatform/magic-modules/pull/6872/commits/ecf83d6134db5a328522cc87faa8e623af763377)